package.json

```diff
@@ -86,10 +89,14 @@
     "url-loader": "^1.1.2",
     "webpack": "~4.47.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-dev-server": "^3.1.14",
+    "yargs": "^13.3.2"
```

Comment:
Since yargs 17.7.2 seemed to work in #669, can you bump yargs to that version here since you are making the effort of upgrading?

Reply:
Upgraded to v17. The only change needed was to how `argv.version` was being handled, but I think I've fixed all the cases on that one.